src/mlpack/tests/load_save_test.cpp

```diff
@@ -1414,8 +1414,19 @@ TEST_CASE("RegularCSVDatasetInfoLoad", "[LoadSaveTest]")
   {
     arma::mat one, two;
     DatasetInfo info;
-    data::Load(testFiles[i], one);
-    data::Load(testFiles[i], two, info);
+  if (!data::Load(testFiles[i], one)) 
```

Comment:
Lines 1417-1429 need to indented another two spaces to maintain the indentation of the block.